RIPS/rip-7712.md

```diff
@@ -60,6 +60,15 @@ Notice that this also prevents the previously signed transactions from becoming
 The old `nonce` account parameter remains in use for transactions initiated by EOAs and for the `CREATE` opcode.
 It is not incremented when `sender` makes the `AA_TX_TYPE` transaction.
 
+### Deployment transaction
+
+As [EIP-161](https://eips.ethereum.org/EIPS/eip-161) defines the rule by which the old `nonce` parameter of a contract
```

Comment:
I don't understand this paragraph.

When deploying an account, the current nonce (in the transaction) has to be zero, regardless of the key (either zero or other key)
after the transaction, the nonce is 1 (for that key).

yes, technically this nonce increment is done by the actual deploy operation, and no "account-creation-specific" logic is done in geth.

Reply:
Please read EIP-161. This is a crucial piece of information.

Reply:
- prior creation, account nonce is always zero (by definition). 
- after deployment , acocunt `nonce` is 1 (since it was created with CREATE/CREATE2)
- if nonceKey is non-zero, we should either change ALSO the nonceKey-based nonce to 1 (2 storage changes), or leave it at zero (which is a bit counter-intuitive: why nonceKey was specified and not incremented.
- maybe the best way is say that during account creation, nonceKey MUST be zero: nonceKey is for "parallelization" anyway, and there can't be any parallelization during account creation.

Reply:
@drortirosh I tried to rephrase the paragraph to make it even more clear. If you think it is still not clear, please propose your own PR. Thanks!